### DISCUSSION TITLE
Repositorio

### DISCUSSION THREAD

**Discussion by cmjosueh** [Launch-X-Latam/MisionFrontEnd#492] [Team Posts]
soy yo o brujeriaTech no a activado el repositorio en donde viene el temario

**Comment by judithbolanos** [comment]
Según yo es este [https://github.com/Launch-X-Latam/MisionBackend](https://github.com/Launch-X-Latam/MisionBackend)

**Comment by kmn04MX** [comment]
¿Dónde obtuviste el enlace?
